fs/zbd_zenfs.cc

```diff
@@ -358,6 +358,10 @@ void ZonedBlockDevice::LogGarbageInfo() {
 
     double garbage_rate =
         double(z->wp_ - z->start_ - z->used_capacity_) / z->max_capacity_;
+    if (z->IsFull()) {
```

Comment:
Please change the code to do the garbage calculation only once (using IsFull() as a condition for how to do it)